dev/ci/ci-basic-overlay.sh

```diff
@@ -69,7 +69,7 @@
 ########################################################################
 # HoTT
 ########################################################################
-: "${HoTT_CI_REF:=ed714382f980d54c055890f95bef0de4ff6bdc78}"
+: "${HoTT_CI_REF:=95aec99612b12747555534e444a0d7274dbcaaf4}"
```

Comment:
Have you tested that this version is still compatible with Coq 8.10?

Reply:
Yes, it builds and I also did some additional tests in CoqIDE, so I have no reason to assume that this version is worse than another version. I will get additional feedback from people giving lectures on the HoTT summer school soon.

Please note that on the 810 branch full Windows installers are enabled now.